packages/broker/src/plugins/testnetMiner/config.schema.json

```diff
@@ -0,0 +1,33 @@
+{
+    "$id": "config.schema.json",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "description": "Testnet miner plugin configuration",
+    "required": [
+        "rewardStreamId",
+        "claimServerUrl",
+        "maxClaimDelay"
+    ],
+    "additionalProperties": false,
+    "properties": {
+        "rewardStreamId": {
+            "type": "string",
+            "description": "streamr.eth/testnets/brubeck-rewards"
```

Comment:
I wonder if you can use keyword "default" here, see: https://json-schema.org/understanding-json-schema/reference/generic.html

Reply:
Fixed